golemfactory/ya-httpx-client
# requestor_proxy example

This PR resolves following JIRA ticket

 
- [APPS-210](https://golemproject.atlassian.net/browse/APPS-210)


## Description

Requestor agent as a proxy to a server(s) running on the provider(s).
This is an example usage of `ya-httpx-client`, but could be useful as a standalone "product". 

## Testing instructions

What we want: working `requestor_proxy`, described as well as possible.
Reader should understand how to run it, how to modify it, and more-or-less how it works.

So: see `README`, try to do something with it.
